collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -57,40 +57,49 @@ abstract class EntitiesRepositoryTest {
     }
 
     @Test
-    fun `save() updates existing entity with matching id and version`() {
+    fun `#save updates existing entity with matching id and version`() {
         val repository = buildSubject()
 
         val wine = Entity("wines", "1", "Léoville Barton 2008", version = 1)
         repository.save(wine)
 
-        val updatedWine = wine.copy(label = "Léoville Barton 2009")
+        val updatedWine = wine.copy(label = "Léoville Barton 2009", version = 1)
```

Comment:
You don't have to set the version number using copy if it's the same. Just the label has changed.